server/server_test.go

```diff
@@ -228,10 +230,5 @@ func (s *testServerHandlerSuite) TestRegisterServerHandler(c *C) {
 	resp, err := http.Get(fmt.Sprintf("%s/pd/apis/mok/v1/hello", svr.GetAddr()))
 	c.Assert(err, IsNil)
 	c.Assert(resp.StatusCode, Equals, http.StatusOK)
-	c.Assert(err, IsNil)
 	defer resp.Body.Close()
-	bodyBytes, err := io.ReadAll(resp.Body)
-	c.Assert(err, IsNil)
-	bodyString := string(bodyBytes)
-	c.Assert(bodyString, Equals, "Hello World\n")
```

Comment:
why delete these lines?

Reply:
according to https://github.com/tikv/pd/pull/4495#discussion_r774458675
But seems it should be reserved. Because it should not break the original test